modules/ROOT/pages/kafka/kafka-connector-studio.adoc

```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In Anypoint Studio, add Kafka Connector to a Mule project, configure the connection to the Kafka cluster, and configure an input source for the connector. 
+In Anypoint Studio, add Kafka Connector to a Mule project, configure the connection to the Kafka cluster, and configure an input source for the connector.
```

Comment:
Have we already used the full connector name?
(It never hurts to repeat it, however: up to you!)